yaksh/models.py

```diff
@@ -1148,11 +1148,14 @@ class TestCase(models.Model):
 class StandardTestCase(TestCase):
     test_case = models.TextField()
     weight = models.FloatField(default=1.0)
+    test_case_args = models.TextField(help_text="<b>Command Line arguments for bash only</b>",
+                                      blank=True)
```

Comment:
I have added a field for command line arguments in `StandardTestCase` for bash.
I am not sure whether to add it in `StandardTestCase` or create a new Test Case.